apps/sms/js/thread_ui.js

```diff
@@ -2513,17 +2512,18 @@ var ThreadUI = {
 
     if (event.target.isPlaceholder) {
       typed = event.target.textContent.trim();
-      this.searchContact(typed, this.listContacts.bind(this));
+      this.searchContact(typed)
```

Comment:
this works in this case, but not "generally" if we need to add more .then handlers. But I can do this here :)